pkg/test/resource_creator_test.go

```diff
@@ -33,15 +33,16 @@ func TestGetOperatorNamespace(t *testing.T) {
 		KubeClient:        fake.NewSimpleClientset(),
 	}
 
-	t.Run("should create a non-empty new Operator Namespace when OperatorNamespace env is not set", func(t *testing.T) {
-		Global.OperatorNamespace = ""
-		ctx := NewContext(t)
-		operatorNamespace, err := ctx.GetOperatorNamespace()
-		assertNoError(t, err)
-		if len(operatorNamespace) <= 0 {
-			t.Errorf("expected non-empty operatorNamespace")
-		}
-	})
+	t.Run("should create a non-empty new Operator Namespace when OperatorNamespace env is not set",
+		func(t *testing.T) {
```

Comment:
Just change the indentation? 👍

Reply:
@camilamacedo86 the `func` is intented like this to preserve the `80 character` lint rule.
i have kept the intendation consistent in other test cases.

Reply:
the indendation is automatically done by `go fmt`